pkgs/development/python-modules/durationpy/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "durationpy";
+  version = "0.5";
+
+  src = fetchPypi {
+    inherit version pname;
+    sha256 = "sha256-XvlBa1J7UNci80ZVvs+3Xkkijrgvh7hV7RkRszFLVAg=";
+  };
+
+  meta = with lib; {
+    description = "Module for converting between datetime.timedelta and Go's Duration strings";
+    homepage = "https://github.com/icholy/durationpy";
+    license = licenses.mit;
+  };
```

Comment:
`meta.maintainers` should be set

Reply:
Yeah. Anyone feel up for it? I really don't know my way around the python ecosystem and don't feel comfortable putting myself in there.

Reply:
This is also missing a pythonImportsCheck

Reply:
@Lurkki14 I added myself as a maintainer.

@SuperSandro2000 I added the check.